docs/en/02_Developer_Guides/14_Files/03_File_Security.md

```diff
@@ -352,11 +359,70 @@ SilverStripe\Filesystem\Flysystem\FlysystemAssetStore:
     Pragma: 'no-cache'
 ```
 
+### Configuring file HTTP response code
+
+When a user tries to access a file that exists, but for which they do not have access,
+Silverstripe CMS will return a "404 Not found" response rather than a "403 Denied" to
+avoid revealing the existence of the file.
+
+You can customise the response codes for various types of request via
+configuration flags on `FlysystemAssetStore`.
+
+```yml
+# Add this code snippet to your YML configuration
+SilverStripe\Filesystem\Flysystem\FlysystemAssetStore:
+  denied_response_code: 403 # This default to 404
+  missing_response_code: 404
+  redirect_response_code: 302
+  permanent_redirect_response_code: 301
+```
+
+### Updating a file HTTP response before it's sent back to the browser
+
+`silverstripe/assets` 1.6 and above allows you to intercept the file HTTP response
```

Comment:
Need to make sure we update this line if the related PR doesn't get merged in time for the 4.6 release.

Reply:
It's merged